Week3/homework/2-step3.js

```diff
@@ -1,8 +1,32 @@
-'use strict';
+"use strict";
 
+// 3.2 make array
 function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
   const values = [];
-  // Add your code here
+  // make array
+  // start at beginning of array and check if you should call
+  // threeCallback or fiveCallback or go on to next
+
+  for (let i = startIndex; i <= stopIndex; i++) {
+    values.push(i);
+  }
+  console.log(values);
+
+  values.forEach(num => {
+    if (num % 3 === 0) {
+      threeCallback(num);
+    }
+    if (num % 5 === 0);
+    {
```

Comment:
the Brackets `{...}` should be in the same line of the `if` statement plus the `if` statement doesn't have a semicolon